tools/frb_internal/lib/src/makefile_dart/misc.dart

```diff
@@ -82,6 +82,12 @@ Future<void> precommitGenerate() async {
     for (final package in kDartExamplePackages)
       generateRunFrbCodegenCommandGenerate(GeneratePackageConfig(
           setExitIfChanged: false, package: package, coverage: false)),
+    for (final package in kDartExampleTestFeaturePackages)
+      generateRunFrbCodegenCommandGenerate(GeneratePackageConfig(
```

Comment:
what about this: Do not add a `GeneratePackageConfig.features` flag (since when we are running this codegen test, we *always* want to have that feature enabled). Instead, inside generateRunFrbCodegenCommandGenerate, maybe we just check if package == pure_dart/pure_dart_pde, and add feature flags accordingly